server/db/seed.js

```diff
@@ -0,0 +1,72 @@
+const seedProducts = [
```

Comment:
I think this file should be removed now that we have the restaurant seed data split out in multiple files

Reply:
Erin is not available for the next few hours, so I deleted the seed.js file.  You can approve the PR and do the rebase for your branch.  I will also rebase for the cart branch.  I need this code for working on the cart.  Thanks.